src/care/host_device_ptr.h

```diff
@@ -409,10 +409,10 @@ namespace care {
    /// @return the output stream after printing
    /////////////////////////////////////////////////////////////////////////////////
    template<typename T>
-   std::ostream& operator<<(std::ostream& os, const host_device_ptr<T>& ptr) {
+   std::ostream& operator<<(std::ostream& os, host_device_ptr<T> ptr) {
```

Comment:
Add a TODO comment here saying that this can be reverted when we have a const version of these functions in chai

Reply:
Done.